app/handlers/personal/start.py

```diff
@@ -169,4 +177,6 @@ def register_handler_start(dp: Dispatcher):
         select_target_language, FSMChooseLanguage.target_language
     )
     dp.callback_query.register(add_words_to_study, ToStudyCallbackData.filter())
+    dp.callback_query.register(my_variant, F.data == 'my_variant')
+    dp.callback_query.register(nothing_to_do, F.data == 'nothing_to_do')
     dp.message.register(translate_text)  # F.test.regexp("[a-zA-Z ]"))
```

Comment:
In fact, I have no comments on the essence of the code, but there are comments on the form. The code does not pass type checks (I indicated in two places, but in total, in one form or another, the linter showed me 35 errors in start.py). Basically, these are simple errors that are easy to fix.
I suggest making this a rule.
I think that it is necessary to include automatic mypy check in the CI process, but after correcting all the errors in the code. Just a separate task for the next sprint.